### PR TITLE
Improve documentation of cms constants (fixes #335)

### LINKED ISSUE #335: Improve documentation of cms constants
Add docstrings of the following format:
```
"""
[Summary]

:param [ParamName]: [ParamDescription], defaults to [DefaultParamVal]
:type [ParamName]: [ParamType](, optional)
...
:raises [ErrorType]: [ErrorDescription]
...
:return: [ReturnDescription]
:rtype: [ReturnType]
"""
```

### PR DESCRIPTION
**Pull request by timoludwig**
- Add docstrings to the cms.constants module
- Add docstrings to the api.v3.feedback module
- Move module description before submodules

### CODE REVIEW

File: src/api/v3/feedback.py [hunk 1]
```diff
@@ -6,6 +6,23 @@
 
 
 class FeedbackData:
+    """
+    This class generic api representation of cms feedback incl. its meta information stored in the database.
```
Comment (Thomas125) [inline src/api/v3/feedback.py]: I think a verb is missing in this sentence.

File: src/cms/constants/administrative_division.py [hunk 2]
```diff
@@ -1,3 +1,35 @@
...
+
+* ``FREE_STATE``: Free state
+
+* ``FEDERAL_STATE``: Federal state
+
+* ``MUNICIPALITY``: Municipality
+
+* ``COLLECTIVE_MUNICIPALITY``: Collective municipality
+
+* ``INITIAL_RECEPTION_CENTER``: Initial reception center
```
Comment (Thomas125) [inline src/cms/constants/administrative_division.py]: Wow, these are really informative descriptions :stuck_out_tongue_closed_eyes:

File: src/api/v3/feedback.py (hunk 1 above)
Comment (timoludwig) [reply src/api/v3/feedback.py]: True!

File: src/cms/constants/administrative_division.py (hunk 2 above)
Comment (timoludwig) [reply src/cms/constants/administrative_division.py]: Do you think the equivalent terms for Germany would help here?

**Pull request merged**